Lesson5/src/come/work/home/model/Military.java

```diff
@@ -25,6 +26,21 @@ public void infoAirMilitary() {
                 "\n missiles :" + getMissiles() + " \n ejection system " + ejectionSystem);
     }
 
+    public boolean controlEjection(){
```

Comment:
clean up code formatting here, please

Reply:
code formatting is still wrong

Reply:
if( ejectionSystem == true){ ->  if (ejectionSystem == true) {
 
 use ctrl + alt + l to fix it